pkg/odo/cli/list/namespace/namespace.go

```diff
@@ -57,15 +58,38 @@ func (plo *NamespaceListOptions) Validate(ctx context.Context) (err error) {
 }
 
 // Run contains the logic for the odo list project command
-func (plo *NamespaceListOptions) Run(ctx context.Context) error {
-	namespaces, err := plo.clientset.ProjectClient.List()
+func (plo *NamespaceListOptions) Run(_ context.Context) error {
+	namespaces, err := plo.run()
 	if err != nil {
 		return err
 	}
 
 	return HumanReadableOutput(os.Stdout, namespaces, plo.commandName)
 }
 
+func (plo *NamespaceListOptions) run() (api.ResourcesList, error) {
+	namespaces, err := plo.clientset.ProjectClient.List()
+	if err != nil {
+		return api.ResourcesList{}, err
+	}
+
+	if len(namespaces.Items) == 0 {
+		return api.ResourcesList{}, fmt.Errorf("you are not a member of any %[1]ss. You can request a %[1]s to be created using the `odo create %[1]s <%[1]s_name>` command", plo.commandName)
```

Comment:
I was testing this, and noticed that the error message displayed here can be a bit confusing:

```
$ odo list namespaces
 ✗  you are not a member of any s. You can request a  to be created using the `odo create  <_name>` command

```

It looks like `plo.commandName` is not being set. I guess we'll need to do something similar to what is done in `pkg/odo/cli/create/namespace/namespace.go`:  https://github.com/redhat-developer/odo/blob/f96c35ac0ee2970eb873bacd2b8af097f4be2110/pkg/odo/cli/create/namespace/namespace.go#L116-L120

Reply:
Also, even with the changes to set `plo.commandName`, I think we might still have an issue with the wording of the error message if users make use of the plural form of the alias, given the string format (`%[1]ss`):

```
x odo list projects
 ✗  you are not a member of any projectss. You can request a projects to be created using the `odo create projects <projects_name>` command
```

Reply:
@rm3l thanks for catching this and for the tip to fix it. I have pushed the fix.